doc/api_assets/runkit.js

```diff
@@ -0,0 +1,33 @@
+(function ()
```

Comment:
can we make sure that this script and others are still using our eslint rules?

Reply:
Sure thing!